Content.Server/Power/Generation/Teg/TegSystem.cs

```diff
@@ -181,7 +181,12 @@ private void GeneratorUpdate(EntityUid uid, TegGeneratorComponent component, ref
         // Turn energy (at atmos tick rate) into wattage.
         var power = electricalEnergy * _atmosphere.AtmosTickRate;
         // Add ramp factor. This magics slight power into existence, but allows us to ramp up.
-        supplier.MaxSupply = power * component.RampFactor;
+        power *= component.RampFactor;
+
+        // Simulate TEG powering itself after being started up. This means that if LV is lost this keeps running.
+        const float load = 1000;
```

Comment:
This should not hard code the power value of the TEG, it should be based on the prototype. Now if somebody changes the value in the YML it won't work.

Reply:
are you going to fix this or should i